### DISCUSSION TITLE
How to make slip indicator

### DISCUSSION THREAD

**Discussion by Informatyk1234** [JSBSim-Team/jsbsim#1198] [Support requests]
Hello there,
I'm trying to make cessna 172 instrument panel working in unreal engine, but I have no idea how to make slip indicator.
![image](https://github.com/user-attachments/assets/2766475a-ef96-43d7-8ef4-a9c3d03a0f33)

**Comment by bcoconni** [comment]
After a quick research on the internet, it seems that the ball is reacting to the lateral acceleration of the aircraft so you should be able to obtain some plausible behavior using the property `accelerations/a-pilot-y-ft_sec2` or calling `FGAuxiliary::GetPilotAccel(eY)` in C++.
